### ISSUE TITLE
comment-no-empty

### ISSUE THREAD

**Issue by evilebottnawi** [stylelint/stylelint#1677]
Disallow empty comments.

Invalid:
```css
/* */
div {
  /* */
  color: red;
  /* */
}
/* */
```

Valid:
```css
/* Some comment */
div {
  /* Some comment */
  color: red;
 /* display: block; */
 /* Some comment */
}
/* Some comment */
```

**Comment by jeddy3** [comment]
Sgtm. Would be aligned with our `block-no-empty` rule. Any objections or shall I label it up?

**Comment by ntwb** [comment]
SGTM

**Comment by davidtheclark** [comment]
No objections.

**Comment by sendilkumarn** [comment]
I will take this one :)

**Comment by evilebottnawi** [comment]
@sendilkumarn let's rock :+1:
